autoarray/inversion/inversion/settings.py

```diff
@@ -7,7 +7,7 @@
 class SettingsInversion:
     def __init__(
         self,
-        use_w_tilde: bool = True,
+        use_w_tilde: bool = False,

```

Comment:
This should be True, I guess you swapped it for testing purposes.

If we have HPC runs going, this could mean they are not using w_tilde and therefore are running slow.